packages/admin/admin/src/mui/menu/ItemGroup.tsx

```diff
@@ -1,31 +1,93 @@
-import { Box, ComponentsOverrides, Theme, Typography } from "@mui/material";
+import { Box, ComponentsOverrides, Theme, Tooltip, Typography } from "@mui/material";
 import { createStyles, WithStyles, withStyles } from "@mui/styles";
+import clsx from "clsx";
 import * as React from "react";
+import { FormattedMessage, MessageDescriptor, useIntl } from "react-intl";
 
-export type MenuItemGroupClassKey = "root" | "title";
+import { MenuContext } from "./Context";
+
+export type MenuItemGroupClassKey = "root" | "title" | "titleMenuOpen" | "titleContainer" | "titleContainerMenuOpen";
 
 const styles = (theme: Theme) =>
     createStyles<MenuItemGroupClassKey, MenuItemGroupProps>({
         root: { marginTop: theme.spacing(8) },
         title: {
             fontWeight: theme.typography.fontWeightBold,
-            fontSize: 14,
+            fontSize: 12,
+            border: `2px solid ${theme.palette.grey[100]}`,
+            borderRadius: 20,
+            padding: theme.spacing(0.5, 2),
             lineHeight: "20px",
+            color: `${theme.palette.grey[300]}`,
+        },
+        titleMenuOpen: {
+            fontSize: 14,
+            border: `2px solid ${theme.palette.common.white}`,
+            borderRadius: "initial",
+            padding: 0,
+            color: theme.palette.common.black,
+        },
+        titleContainer: {
             borderBottom: `1px solid ${theme.palette.grey[50]}`,
+            display: "flex",
+            justifyContent: "center",
+            padding: `${theme.spacing(2)} 0`,
+        },
+        titleContainerMenuOpen: {
+            justifyContent: "flex-start",
             padding: theme.spacing(2, 4),
         },
     });
 
 export interface MenuItemGroupProps {
-    title?: React.ReactNode;
+    title: React.ReactNode;
+    shortTitle?: string;
```

Comment:
```suggestion
    shortTitle?: React.ReactNode;
```

We could still support ReactNode here so we can use `FormattedMessage` in applications